bundles/org.openhab.binding.heliosventilation/src/main/resources/ESH-INF/i18n/heliosventilation_de.properties

```diff
@@ -0,0 +1,79 @@
+# binding
```

Comment:
Nice. Tahnks for providing i18n

Reply:
That was quite selfish, as I want to read the things in my mother tongue :smile: